ed25519-dalek/src/verifying.rs

```diff
@@ -505,6 +505,11 @@ impl VerifyingKey {
     pub fn to_montgomery(&self) -> MontgomeryPoint {
         self.point.to_montgomery()
     }
+
+    /// Return this verifying key in Edwards form.
+    pub fn to_edwards(&self) -> EdwardsPoint {
+        self.point
+    }
```

Comment:
Perhaps add a `From` impl too?

Reply:
There is already a `impl From<EdwardsPoint> for VerifyingKey` - is your suggestion here to add the cooresponding `impl From<VerifyingKey> for EdwardsPoint`?

Reply:
Done